hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestUnclosedIO.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.hudi.cli.commands;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class TestUnclosedIO {
```

Comment:
We need to test the `RepairsComand` and `TableCommand` can work as expected with the fix, or a manual test should be good too.

Reply:
emm! the RepairtsCommand tests passed,but the I don't know how to test the TableCommand,Their scenarios are the same.

Reply:
emm!All tests passed！